app/models/diff.js

```diff
@@ -4,6 +4,9 @@ var comment = /^\\/,
     deleted = /^\-/,
     inserted = /^\+/;
 
+const targetFileBaseUrl =
+  'https://raw.githubusercontent.com/railsdiff/api/master/generated/';
```

Comment:
I would prefer to use `https://github.com/railsdiff/api/raw/master/generated/` here, I know it'll cause a redirect to this host, but it seems a little more future-proof.